### PR TITLE
Add Admission and Pupil Benefits config as part of onboarding

### PR DESCRIPTION
**Pull request by bella-cockrell**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/DOC-685

### *What is the problem we're trying to solve*

Adding config for Admission and Pupil Benefits (School's Admissions).

### *What changes have we introduced*

- Added document types
- Added staff-selected document types

#### _Checklist_

- [ ] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [ ] Added tests to cover all new production code
- [ ] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [ ] Checked all code for possible refactoring
- [ ] Code pipeline builds correctly

### *Follow up actions after merging PR*

Are there any next steps that need to addressed after merging this PR? Add them here.

### CODE REVIEW

File: EvidenceApi/DocumentTypes.json [hunk 1]
```diff
@@ -211,5 +211,31 @@
...
+        "description": "Two valid documents to provide proof of address. It must inculde the name of the Parent who is applying clearly wittren on the document provided."
+      },
+      {
+        "id": "proof-of-child-dob",
+        "title": "Proof of Child's Date of Birth",
+        "description": "A valid document to provide proof of child's identity. Picture of child's full name and date of birth must be in the Evidence."
+      },
+      {
+        "id": "proof-of-arrival",
+        "title": "Proof of Child's Date of Birth",
```
Comment (andrewclowes) [inline EvidenceApi/DocumentTypes.json:231]: Should this be `Proof of Child's Date of Arrival`?

File: EvidenceApi/DocumentTypes.json [hunk 2]
```diff
@@ -211,5 +211,31 @@
...
     ]
+  },
+  {
+    "name": "Admission and Pupil Benefits",
+    "id": "9",
+    "documentTypes": [
+      {
+        "id": "proof-of-address",
+        "title": "Proof of Address",
+        "description": "Two valid documents to provide proof of address. It must inculde the name of the Parent who is applying clearly wittren on the document provided."
```
Comment (andrewclowes) [inline EvidenceApi/DocumentTypes.json:222]: `written`?

File: EvidenceApi/DocumentTypes.json (hunk 1 above)
Comment (bella-cockrell) [reply EvidenceApi/DocumentTypes.json]: I've updated this to `Proof of Arrival`, as it is in the spreadsheet.

File: EvidenceApi/DocumentTypes.json (hunk 2 above)
Comment (bella-cockrell) [reply EvidenceApi/DocumentTypes.json]: Good spot! Fixed the typo

**Review by andrewclowes — APPROVED** [review]

**Pull request merged**